class CustomException(Exception):
    pass


class StudentNotFoundException(CustomException):
    pass
